src/handle_finalize.c

```diff
@@ -8,9 +8,21 @@ void handle_finalize(void *parameters) {
         case PURCHASE:
             msg->numScreens = 1;
             break;
+        case PURCHASE_HOLDER:
+            msg->numScreens = 1;
+            break;
+        case PURCHASE_MERKLE:
+            msg->numScreens = 1;
```

Comment:
Should these not be 2 (and 3 below) or am I misunderstanding the idea that they are derived from # of input purchase params?

Reply:
For the Merkle minter, there isn't really an easy way to display the proof in a format that makes sense to the user - for the token holder minter, I can add the address and token ID yup.

Reply:
Can you take another look? The main change previously was adding the new function selectors, but I just pushed changes that add the token holder and merkle proof params.